src/qml/util.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <qml/util.h>
+
+#include <string>
+
+#include <QQuickWindow>
+#include <QSGRendererInterface>
+#include <QString>
+
+namespace QmlUtil {
+
+QString GraphicsApi(QQuickWindow* window)
+{
+    switch (window->rendererInterface()->graphicsApi()) {
+    case QSGRendererInterface::Unknown: return "Unknown";
```

Comment:
I've spent hours and hours digging through Qt code to figure out if there is a better way than manually maintaining these strings. Sadly, there is not. For now, this allows us to conveniently debug. 

Note: Let's revisit this at some point in the future